# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1293514735 28800
#      Mon Dec 27 21:38:55 2010 -0800
# Node ID 1a7b41db5432559f4af4b2ac0232cd8b2176f664
# Parent  74e285bb61c958dc153bc12e9aabd2e45b1d424f
Add additional calling form using file descriptor (fid) to docstring for stat.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-12-27  Rik  <octave@nomad.inbox5.com>
+
+	* syscalls.cc (stat): Add additional calling form using file descriptor
+	fid to docstring.
+
 2010-12-26  Michael Godfrey <godfrey@isl.stanford.edu>
 
 	* gammainc.cc: Add @tex blocks to docstring.
 
 2010-12-25  Rik  <octave@nomad.inbox5.com>
 
 	* DLD-FUNCTIONS/str2double.cc: Fix bug in converting "numberi" strings
 	such as "2i".  Add ability to process special value "NA".  Rewrite 
diff --git a/src/syscalls.cc b/src/syscalls.cc
--- a/src/syscalls.cc
+++ b/src/syscalls.cc
@@ -873,19 +873,21 @@ system-dependent error message.\n\
     print_usage ();
 
   return retval;
 }
 
 DEFUNX ("stat", Fstat, args, ,
   "-*- texinfo -*-\n\
 @deftypefn  {Built-in Function} {[@var{info}, @var{err}, @var{msg}] =} stat (@var{file})\n\
+@deftypefnx {Built-in Function} {[@var{info}, @var{err}, @var{msg}] =} stat (@var{fid})\n\
 @deftypefnx {Built-in Function} {[@var{info}, @var{err}, @var{msg}] =} lstat (@var{file})\n\
+@deftypefnx {Built-in Function} {[@var{info}, @var{err}, @var{msg}] =} lstat (@var{fid})\n\
 Return a structure @var{s} containing the following information about\n\
-@var{file}.\n\
+@var{file} or file identifier @var{fid}.\n\
 \n\
 @table @code\n\
 @item dev\n\
 ID of device containing a directory entry for this file.\n\
 \n\
 @item ino\n\
 File number of the file.\n\
 \n\
